src/FluentValidation.Tests/ValidateAndThrowTester.cs

```diff
@@ -38,7 +38,16 @@ public class ValidateAndThrowTester {
 			typeof(ValidationException).ShouldBeThrownBy(() => validator.ValidateAndThrow(new Person()));
 		}
 
-		[Fact]
+        [Fact]
+        public void Throws_exception_with_a_ruleset() {
+            var validator = new TestValidatorWithRuleSet {

```

Comment:
I'd prefer to use `TestValidatorWithRuleSet` instead of `TestValidatorWithRuleSet` and inline add the `RuleSet`. 

Do you know how I could do that (so I can update this file) ?